docs/api.md

```
@@ -38,21 +38,13 @@ Simple-PHP-IPAM exposes a JSON REST API (`api.php`). Read endpoints are availabl
 
 ## Authentication
 
-Every request must include a valid API key. Two methods are supported:
-
-### Authorization header (recommended)
+Every request must include a valid API key via the `Authorization` header:
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)